Makefile.include

```diff
@@ -327,6 +327,13 @@ ifneq (, $(filter clean, $(MAKECMDGOALS)))
     all $(BASELIBS) $(USEPKG:%=$(RIOTPKG)/%/Makefile.include) $(RIOTBUILD_CONFIG_HEADER_C): clean
 endif
 
+# include Makefile.includes for packages in $(USEPKG)
+$(RIOTBASE)/drivers/%/Makefile.include::
+	$(Q)"$(MAKE)" -C $(RIOTBASE)/drivers/$* Makefile.include
+
+.PHONY: $(USEMODULE:%=$(RIOTBASE)/drivers/%/Makefile.include)
```

Comment:
This is a remaining mistake from packages that should be removed.
I just did not find an opportunity to remove it without only saying "it's bad".

We should never generate files in the source directory and Makefile.include should be BOARD/build configuration independent so not have any reasons to be generated at compile time.